### ISSUE TITLE
Feature request: Automatically stop generation

### ISSUE THREAD

**Issue by TrajansRow** [alexrozanski/LlamaChat#23]
It would be nice to have an option to automatically stop generation after a random number of 1-n sentences, delimited by punctuation followed by whitespace. Additionally, it may be helpful to have a "Stop generation at end of sentence" button as an alternative to the current control in the chat window.

**Comment by TrajansRow** [comment]
Change to this request: Simply expose the reverse-prompt input, which in combination with #17 should provide a versatile method of controlling generation.

**Comment by alexrozanski** [comment]
@TrajansRow yep good shout. Prompt context is something I want to tackle for the next version so will add it to the list
